components/Aside.js

```diff
@@ -17,17 +17,15 @@ const FileList = ({ title, files, packageName }) => html`
       x => html`
         <li key=${x.url} data-test="Item">
           ${FileIcon}
-          <a
+          <${Link}
             href=${x.url.replace('https://unpkg.com/', '/?')}
-            onClick=${e => {
-              if (shouldPreventDefault(e)) {
-                e.preventDefault();
-                pushState(`?${x.url.replace('https://unpkg.com/', '')}`);
-              }
+            onClick=${() => {
```

Comment:
I thought the idea of making a Link component is so that you don't need this `onClick` handler? you should just be able to provide an href and the component handle the navigation as standard.

Reply:
Yep. Silly oversight by me!